versions.yaml

```diff
@@ -149,6 +149,7 @@ module-sets:
       - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudspannerreceiver
       - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver
       - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver
+      - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver
```

Comment:
I think leave this out for now.

Reply:
There was a check that required it in github actions but maybe there's a label that ignores it?